h2o-automl/src/main/java/ai/h2o/automl/AutoML.java

```diff
@@ -914,7 +914,7 @@ void defaultGBMs(Key<Grid> gridKey) {
     return dlJob;
   }
 
-  Job<StackedEnsembleModel>stack(Key<Model>[]... modelKeyArrays) {
+  Job<StackedEnsembleModel>stack(boolean bestModels, Key<Model>[]... modelKeyArrays) {
```

Comment:
The boolean flag doesn't have any actual purpose. I would directly pass modelName instead of the flag - this is clearer API to the user.

Reply:
Yes, I can pass in `String modelName`